app/renderer/components/navigation/urlBarIcon.js

```diff
@@ -111,7 +111,9 @@ class UrlBarIcon extends React.Component {
 
   onDragStart (e) {
     dndData.setupDataTransferURL(e.dataTransfer, this.props.location, this.props.title)
-    dndData.setupDataTransferBraveData(e.dataTransfer, dragTypes.TAB, this.props.activeFrame)
+    dndData.setupDataTransferBraveData(e.dataTransfer, dragTypes.TAB, {
+      activeFrameKey: this.props.activeFrameKey
```

Comment:
sorry to do this to you, but can we use `tabId` instead of `frameKey`? If we had the `tabId` it would be easy to convert this to a browser process action (please add TODO for that) with a `siteUtil.getDetailFromTabId` method. I think we should drop `active` as well once it gets to this point